src/Functions/test/Functions.Scenario.Tests.ps1

```diff
@@ -810,15 +877,15 @@ Describe 'Functions End to End Tests' {
         }
     }
 
-    It "Validate New-AzFunctionAppPlan -AsJob, New-AzFunctionApp -AsJob and Remove-AzFunctionApp -Force" {
+    It "Validate 'New-AzFunctionAppPlan -AsJob', 'New-AzFunctionApp -AsJob' and 'Remove-AzFunctionApp -Force'" {
 
         $planName = "Func99-Windows-Premium-205029017"
```

Comment:
smae comment - these names can be partially randomized if you define them in the setup script, and then added to the env hashtable for retrieval in the test.  This will make is easier to run the test live in subsequent runs.

Reply:
Fixed.